src/node/index.ts

```diff
@@ -0,0 +1,4 @@
+export function overrideNode() {
+  require("./extensions").overrideNode();
+}
+overrideNode();
```

Comment:
Why the extra function wrapper?

Reply:
Please see https://github.com/entropic-dev/ds/pull/2#discussion_r301457809.